docs/guides/fedora/_include/live-environment.rst

```diff
@@ -1,5 +1,5 @@
-Early Setup
------------
+Configure Live Environment
+--------------------------
```

Comment:
The `os-release` and `zgenhostid` sections should be replaced with

```rst
..include:: ../_include/live-environment.rst
```

Reply:
the ordering of steps between fedora/debian/void means that a generic live-environment.rst can't be used that contains both zgenhostid and sourcing os-release. I've broken those up in to two distinct includes, and they're now sourced in the correct order in the per-distro live-environment.rst.